src/current/_includes/releases/v23.1/v23.1.16.md

```diff
@@ -0,0 +1,17 @@
+## v23.1.16
+
+Release Date: February 27, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-16-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where [rangefeed](https://www.cockroachlabs.com/docs/v23.1/use-changefeeds#enable-rangefeeds) resolved timestamps could get stuck, continually emitting the [log message](https://www.cockroachlabs.com/docs/v23.1/logging) `"pushing old intents failed: range barrier failed, range split"`, typically following a [range merge](https://www.cockroachlabs.com/docs/v23.1/architecture/distribution-layer#range-merges). This bug was introduced in v23.1.15. [#119559][#119559]
+
+<h3 id="v23-1-16-contributors">Contributors</h3>
+
+This release includes 2 merged PRs by 2 authors.
```

Comment:
Is it weird or normal that we name 2 PRs but only have 1 release note?

Reply:
not every merged PR is release-note worthy.  IIRC only a subset of total PRs have a release note annotation due to a public-facing change